src/commands/games.ts

```diff
@@ -9,6 +9,7 @@ import { errorEmbedGenerator } from "../modules/commands/errorEmbedGenerator";
 import { handleFact } from "../modules/commands/subcommands/games/handleFact";
 import { handleHabitica } from "../modules/commands/subcommands/games/handleHabitica";
 import { handleJoke } from "../modules/commands/subcommands/games/handleJoke";
+import { handleQuote } from "../modules/commands/subcommands/games/handleQuote";
 import { handleMtg } from "../modules/commands/subcommands/games/handleMtg";
```

Comment:
```suggestion
import { handleMtg } from "../modules/commands/subcommands/games/handleMtg";
import { handleQuote } from "../modules/commands/subcommands/games/handleQuote";
```

Correct import order.